aspnetcore/security/how-to-choose-identity-solution.md

```diff
@@ -0,0 +1,114 @@
+* Use the resulting credentials to authenticate with other apps without having to sign-in again or choose a different password.
+
+A token server is typically preferred to provide a secure and scalable solution for single sign on. 
+
+For apps that don't expose web-based APIs that are consumed by other apps and don't share logins with other apps, the simplest way to quickly secure an app is to use the built-in ASP.NET Core Identity provider. Otherwise, a token server provided by a third-party identity management solution is needed. Token servers are available as:
+
+* Products you install on your server, called *self-host*.
+* Containers run in a host like Docker.
+* Web-based services you integrate with to manage identity.
+
+Some solutions are free and open source, while others are commercially licensed. See [identity management solutions](xref:security/identity-management-solutions) for a list of available options. It is possible that your organization already uses an existing identity provider. In that case, it may make sense to use the existing provider instead of going with a different solution. All of the major solutions provide documentation for configuring ASP.NET Core to use their product or service.
+
+## Disconnected scenarios
+
+Many solutions, such as [Microsoft Entra ID](/azure/active-directory/fundamentals/active-directory-whatis), are cloud-based and require an Internet connection to work. If your environment doesn't allow Internet connectivity, you won't be able to use the service. If you must support scenarios where either your app cannot access the Internet or must still function on the local network even if the Internet is disconnected, choose one of the following options:
+
+* A solution that allows you to install and run the service on your own machines.
+* Run the authentication service locally as a container.
+
+## Decide where user data such as sign-ins are stored
+
+Another important factor to consider is where the user sign-in data is stored. Many developers choose external, cloud-based services like Microsoft Entra ID to manage identity. A cloud-based service provider:
+
+* Takes on the responsibilities of securely storing data.
+* keeps the software up to date with the latest security patches and releases.
+* Complies with privacy regulations.
+
+Others prefer to store data on their own servers due to regulations, compliance, policy, or other reasons.
+
+If the data is stored on your servers, you'll most likely need to choose an installable or container-based solution.
+
+## Identity vs token server
+
+Use the following diagram to help you decide whether to use the ASP.NET Core Identity system or a token server for authentication and authorization:
+
+![Identity management decision flow](./_static/identity-management-decision-flow.png)
+
+The following table lists some of the things to consider when choosing your identity management solution.
+
+|**Feature**  |**Self-host**  |**Container**  |**Cloud** |
```

Comment:
Is "self-host" exclusively Duende? Everything else would run out of process, right? That would make everything else effectively equivalent to the "Container" column for the purposes of this table.

We should clarify this.